cookbook/console/command_in_controller.rst

```diff
@@ -0,0 +1,85 @@
+.. index::
+   single: Console; Use commands in your Controller
+
+How to make use of a Command in your application
+================================================
+
+The Console page of the Components section (:doc:`/components/console/introduction`) covers
+how to create a console command. This cookbook article covers how to use a console command
+directly in your application.
+
+Execute Command directly from Controller
+----------------------------------------
+
+Symfony allows you to directly execute a ``Command`` inside your Controller::
+
+    // src/AcmeBundle/Controller/GreetController.php
+    namespace AcmeBundle\Controller;
+
+    use Symfony\Component\Console\Output\StreamOutput;
+    use Symfony\Component\Console\Input\ArrayInput;
+    use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+    class GreetController
+    {
+        public function greetAction($name)
+        {
+            $kernel = $this->get('kernel');
+            $application = new Application($kernel);
+            $application->setAutoExit(false);
+
+            $input = new ArrayInput(array('command' => 'acme:greet'));
+            $output = new StreamOutput(tmpfile(), StreamOutput::VERBOSITY_NORMAL);
+            $application->run($input, $output);
+
+            rewind($output->getStream());
+            $content = stream_get_contents($output->getStream());
+            fclose($output->getStream());
+
+            return $content;
+        }
+    }
+
+Color code the Ansi content to HTML
+----------------------------------------
+
+By telling the ``StreamOutput`` it is decorated via the third parameter, it will return the Ansi color-coded content.
```

Comment:
This line is a bit long, could you please wrap it to the usual 80 character length? Thanks.